src/core/ext/transport/chttp2/transport/chttp2_transport.cc

```diff
@@ -605,6 +611,38 @@ static void init_keepalive_pings_if_enabled_locked(
   }
 }
 
+void grpc_chttp2_transport::WriteSecurityFrame(grpc_core::SliceBuffer* data) {
+  combiner->Run(grpc_core::NewClosure([transport = Ref(), data]
+                                      (grpc_error_handle) mutable {
+                  transport->WriteSecurityFrameLocked(data);
+                  transport->Unref();
```

Comment:
transport is a smart pointer and shouldn't need to be explicitly unreffed

Reply:
Thanks, missed that it wasn't a raw pointer.